examples/kubernetes/README.md

```diff
@@ -0,0 +1,36 @@
+# OpenTelemetry Collector Demo
+
+This demo is a sample app to build the collector and exercise its kubernetes logs scrapping functionality.
+
+## Build and Run
+
+Two steps are required to build and run the demo:
+
+1. Build latest docker image in main repository directory `make docker-otelcontribcol`
+1. Switch to this directory and run `docker-compose up`
```

Comment:
Docker compose is fine for developer testing but I think for end users will be more valuable to include k8s config and explain how to try it.

What you have here is good for us developers, so let's keep it too.

Up to you if you want to add k8s config example in this PR or another PR.

Reply:
What do you mean by k8s config example? Example daemonset with configmap? Otelcol configuration is going to be the same like in this example